src/components/Button/Button.tsx

```diff
@@ -10,19 +10,37 @@ const Button: React.FC<ButtonProps> = ({
   href,
   target,
   disabled,
+  variant = "primary",
+  size = "normal",
+  icon: Icon,
 }) => {
   const {s} = useStyles();
+  const [keyDown, setKeyDown] = React.useState(false);
```

Comment:
You can import "useState" from React instead of doing React.useState
It's not blocking but good to know if you have multiple states in a single file